docs/building-experiments/metrics/creating-metrics.md

```diff
@@ -16,15 +16,15 @@ This should be one of the facts that you created in the step above, and should c
 
 ![Select Aggregation](../../../static/img/building-experiments/select-aggregation-for-metric.png)
 
-The aggregation will aggregate over whatever the fact is measuring on a per-entity basis. So for example, if you select the `Revenue` fact and the `SUM` aggregation, the metric will be the total revenue for each user;, if you select the `Upgrades` fact and the `SUM` aggregation, the metric will be the total number of upgrades for each user, which should only be 1 across the board.
+The aggregation will aggregate over whatever the fact is measuring on a per-entity basis. So for example, if you select a `Revenue` fact and the `SUM` aggregation, the metric will be "total revenue per user;" if you select a `Name of Article Viewed` fact and the `COUNT DISTINCT` aggregation, the metric will be "Unique articles viewed per user."
```

Comment:
Should we be clear that we're always taking the average across entities, e.g. "average total revenue per user"

Reply:
Making this change